src/Pages/Collection/Collection.scss

```
@@ -0,0 +1,154 @@
+.Collection {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  header {
+    h1 {
+      color: #8f5b59;
+      font-size: 48px;
+      padding: 80px 0 50px;
+      margin-top: 120px;
+      width: 1300px;
+      font-family: "PT Serif", serif;
+      font-weight: bold;
+      letter-spacing: 1px;
+    }
+  }
+  .collectionContainer {
+    margin-bottom: 100px;
+    .collectionBox {
+      .attitude {
+        display: flex;
+        align-items: center;
```

```suggestion
    }
  }
  .collectionContainer {
    margin-bottom: 100px;
    
    .collectionBox {
      .attitude {
        display: flex;
        align-items: center;
```
셀렉터끼리 구분될 수 있도록 새로운 셀렉터 앞에서는 한줄 띄워주세요